package.json

```diff
@@ -1,22 +1,20 @@
 {
```

Comment:
Definitely a good idea! You're right. Since we don't use npm on the static-site-side it wouldn't really matter, but from a perspective of writing correct code :)